### DISCUSSION TITLE
99.9% accuracy when the images look identical.

### DISCUSSION THREAD

**Discussion by AtharvaNanavate** [cssbattle/forum#77] [Bugs reports]
Battle #8 - Transition / #44 (Stripes) :
The images look identical with the code I have written. But the accuracy is 99.9%.
<img width="951" alt="Target#44 - Stripes" src="https://user-images.githubusercontent.com/85158270/124350109-5e9f0780-dc10-11eb-947a-43f91602d0b7.png">
The following is my code :

[CSSBattle_Stripes.txt](https://github.com/cssbattle/forum/files/6758185/CSSBattle_Stripes.txt)

Please help.

**Comment by SelenIT** [comment]
This target is really tricky to get 100% match. For some reason, not all properties that generate the visually matching shape are accepted. Maybe [this thread](https://github.com/cssbattle/forum/discussions/12) could help.
